tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_media_storage_change.py

```diff
@@ -35,6 +35,9 @@ def test_settings_media_storage_change(self):
         # Apparently, the ability of making a photo is very fragile on low memory devices
         self.apps.kill_all()
 
+        # Workaround for bug 1218115 where the camera doesn't launch without this small pause
+        import time
+        time.sleep(0.5)
         camera = Camera(self.marionette)
         camera.launch()
```

Comment:
That's not going to work, the camera app doesn't launch at all when trying it too quickly after kill_all().
I believe the unit tests use similar workarounds for this:
http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/unit/test_kill_multiple.py#19
http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/unit/test_killall.py#17